featuretests/cmd_juju_model_test.go

```diff
@@ -221,7 +221,7 @@ func (s *cmdModelSuite) TestModelDefaultsResetRegion(c *gc.C) {
 	err := s.State.UpdateModelConfigDefaultValues(map[string]interface{}{"special": "known"}, nil, &environscloudspec.CloudRegionSpec{"dummy", "dummy-region"})
 	c.Assert(err, jc.ErrorIsNil)
 
-	s.run(c, "model-defaults", "dummy-region", "--reset", "special")
+	s.run(c, "model-defaults", "--cloud", "/dummy-region", "--reset", "special")
```

Comment:
Is `/dummy-region` valid? I've only ever seen it fully specified as `cloud/region`. The latter is better as it's explicit.